### Data Collection

The datasets was taken from [WHO](http://www.who.int/gho/tb/en/) which consists of total population and total number of deaths due to TB in 2013 in each of the BRICS (Brazil, Russia, India, China, South Africa) and Portuguese-speaking countries into an Excel file.

Based on the population and number of deaths due to TB in some countries during 2013 the following questions answered:

* What is the total, maximum, minimum and average number of deaths in that year?
* Which countries have the most and the least deaths?
* What is the death rate (deaths per 100,000 inhabitants) for each country?
* Which countries have the lowest and highest death rate?

 

#### Import Module and Data

In [2]:
from pandas import* # To import pandas module..

In [3]:
data = read_excel("WHO POP TB some.xls") # To import data into notebook.. 
data.head() # print data with top five rows its columns..

Country  Population (1000s)  TB deaths
0             Angola               21472       6900
1             Brazil              200362       4400
2              China             1393337      41000
3  Equatorial Guinea                 757         67
4      Guinea-Bissau                1704       1200

#### Analysis by accessing columns:

In [9]:
tb_column = data["TB deaths"]
print("The total number of deaths in 2013 due to TB is:",tb_column.sum()) # Computing the total number of deaths..
print("The largest number of deaths in a single country are:", tb_column.max()) # Computing the largest number of deaths..
print("The smallest number of deaths in a single country are:", tb_column.min()) # Computing the Smallest number of deaths..

The total number of deaths in 2013 due to TB is: 354715
The largest number of deaths in a single country are: 240000
The smallest number of deaths in a single country are: 18


In [11]:
# The average number of deaths..
print("The mean value:",tb_column.mean())
print("The median value:",tb_column.median())

The mean value: 29559.583333333332
The median value: 5650.0


#### The most affected Countries
To see the most affected countries, the table is sorted in ascending order by the last column, which puts those countries in the last rows.

In [12]:
data.sort_values("TB deaths") # TB deaths columns sorted in ascending order...

Country  Population (1000s)  TB deaths
9   Sao Tome and Principe                 193         18
3       Equatorial Guinea                 757         67
7                Portugal               10608        140
11            Timor-Leste                1133        990
4           Guinea-Bissau                1704       1200
1                  Brazil              200362       4400
0                  Angola               21472       6900
8      Russian Federation              142834      17000
6              Mozambique               25834      18000
10           South Africa               52776      25000
2                   China             1393337      41000
5                   India             1252140     240000

The table raises the possibility that a large number of deaths may be partly due to a large population. To compare the countries on an equal footing, the death rate per 100,000 inhabitants is computed.

In [14]:
populationColumn = data['Population (1000s)'] # The death rate per 100,000 inhabitants is computed..
data['TB deaths (per 100,000)'] = tb_column * 100 / populationColumn
data

Country  Population (1000s)  TB deaths  \
0                  Angola               21472       6900   
1                  Brazil              200362       4400   
2                   China             1393337      41000   
3       Equatorial Guinea                 757         67   
4           Guinea-Bissau                1704       1200   
5                   India             1252140     240000   
6              Mozambique               25834      18000   
7                Portugal               10608        140   
8      Russian Federation              142834      17000   
9   Sao Tome and Principe                 193         18   
10           South Africa               52776      25000   
11            Timor-Leste                1133        990   

    TB deaths (per 100,000)  
0                 32.134873  
1                  2.196025  
2                  2.942576  
3                  8.850727  
4                 70.422535  
5                 19.167186  
6                 69.675621  
7                  1.319759  
8                 11.901928  
9                  9.326425  
10                47.370017  
11                87.378641

### Conclusions

The BRICS and Portuguese-speaking countries had a total of about 350 thousand deaths due to TB in 2013. The median shows that half of these coutries had fewer than 5,650 deaths. The much higher mean (29,000 plus) indicates that some countries had a very high number. The **least** affected were Sao Tome and Principe, and Equatorial Guinea, with 18 and 67 deaths respectively, and the **most** affected were China and India with 41 thousand and 240 thousand deaths in a single year. However, taking the population size into account, the least affected were Portugal and Brazil with less than 2.2 deaths per 100 thousand inhabitants, and the most affected were Guinea-Bissau and Timor-Leste with over 70 deaths per 100,000 inhabitants.

It convey the message that TB is still a major cause of fatalities, and that there is a huge disparity between countries, with several ones being highly affected.

